draft-ietf-acme-acme.md

```diff
@@ -303,12 +303,21 @@ JWS objects sent in ACME requests MUST meet the following additional criteria:
 * The JWS MUST NOT have the value "none" in its "alg" field
 * The JWS Protected Header MUST include the following fields:
   * "alg"
-  * "jwk"
+  * "jwk" (only for requests to new-reg and revoke-cert resources)
+  * "kid" (for all requests except for new-reg and revoke-cert requests).
   * "nonce" (defined below)
   * "url" (defined below)
 
-Note that this implies that GET requests are not authenticated.  Servers MUST
-NOT respond to GET requests for resources that might be considered sensitive.
+For new-reg requests and revoke-cert-by-key requests, there MUST be a
+"jwk" field containing the key being registered, and there MUST NOT be a
+"kid" field.
```

Comment:
Adding "revoke-cert-by-key" is too much to glom on to this PR.  Please pull it out and file a separate PR.  I think we can leave the remaining wording as it is (i.e., "jwk" for new-reg, "kid" for *), in anticipation of that follow-on.  No objection to "revoke-cert-by-key" in principle, I just think we might need an RTT or two on how exactly it's specified.